qiita_pet/templates/list_studies.html

```diff
@@ -38,7 +44,7 @@
   moi.send('sel', proc_data);
 }
 
-function show_alert(data) { bootstrapAlert(data + ' samples selected.', "success", 2000); }
+function show_alert(data) { bootstrapAlert(data + ' samples selected.', "success", 2000); $('#dflt-sel-info').css('color', 'rgb(0, 160, 0)');}
```

Comment:
I don't, but javascript doesn't care about line breaks and it saves some transmit info.